docs/reference/contrib/settings.rst

```diff
@@ -205,21 +205,21 @@ Add ``wagtail.contrib.settings.jinja2tags.settings`` extension to your Jinja2 se
 
 Then access the settings through the ``settings()`` template function:
 
-.. code-block:: html+jinja
+.. code-block:: html+django
```

Comment:
`html+jinja` is correct for this section - it's talking about jinja2 template tags.